packages/dcos-integration-test/extra/test_mesos.py

```diff
@@ -76,9 +63,29 @@ def test_if_marathon_app_can_be_debugged(dcos_api_session):
             if agent['id'] == agent_id:
                 agent_hostname = agent['hostname']
         assert agent_hostname is not None, 'Agent hostname not found for agent_id {}'.format(agent_id)
-        agent_v1_url = 'http://{}:{}/api/v1'.format(agent_hostname, 5051)
         logging.debug('Located %s with containerID %s on agent %s', app_id, container_id, agent_hostname)
 
+        # Wrapper to post, log, and validate return status
+        def _post_agent(url, headers, json=None, data=None, stream=False):
+            r = dcos_api_session.post(
+                url,
+                host=agent_hostname,
+                port=5051,
+                headers=headers,
+                json=json,
+                data=data,
+                stream=stream)
+            agent_url = '{}://{}:{}/state'.format(scheme, agent_hostname, 5051)
```

Comment:
But in the case of the current callers of this function, it's `/api/v1` not `/state`, and there's no way to guarantee that's always the case since `url` is a parameter to `_post_agent` and this function can and likely will be reused elsewhere, with different urls. Please log the actual URL being 'post'ed. That was my original intention in these log lines.